pkg/app/app.go

```diff
@@ -164,9 +165,36 @@ func (a *App) Start() {
 		a.log.Notice("Broker successfully bootstrapped on startup")
 	}
 
+	interval, err := time.ParseDuration(a.config.Broker.RefreshInterval)
+	a.log.Debug("RefreshInterval: %v", interval.String())
+	if err != nil {
+		a.log.Error(err.Error())
+		a.log.Error("Not using a refresh interval")
+	} else {
+		ticker := time.NewTicker(interval)
```

Comment:
do we have a minimum interval? I mean how often can we really bootstrap? We should probably figure out what our minimum is and verify that during reading of the config above.